src/webgpu/web_platform/reftests/copy_element_image_to_texture.https.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html class="reftest-wait">
+  <title>WebGPU copyElementImageToTexture</title>
+  <meta charset="utf-8" />
+  <link rel="help" href="https://github.com/WICG/html-in-canvas" />
+  <link rel="match" href="ref/copy_element_image_to_texture-ref.html" />
+
+  <style>
+   canvas {
+     border: 0;
+   }
+   .drawn-element {
+     width: 256px;
+     height: 192px;
+     background-color: white;
+   }
+   .css-is-awesome {
+     width:3.2rem;
+     border:.1rem solid black;
+     border-radius:.5rem;
+     padding:.2rem;
+     background:linear-gradient(45deg,rgb(240 176 176),rgb(176 240 240));
+     font-family:sans-serif;
+   }
+   .css-is-awesome::before {
+     content: "CSS IS AWESOME";
+   }
+  </style>
+
+  <canvas id="cvs" width="256" height="192" layoutsubtree>
+    <div id="target" class="drawn-element">
+      <div class="css-is-awesome"></div>
+    </div>
+  </canvas>
+
+  <script type="module">
+import { runRefTest } from './gpu_ref_test.js';
+import { assert } from '../../../common/util/util.js';
+runRefTest(async (t) => {
+  const cvs = document.getElementById('cvs');
+  const ctx = cvs.getContext('webgpu');
+  assert(ctx instanceof GPUCanvasContext,
+         'Failed to get WebGPU context from canvas');
+
+  t.device.addEventListener('uncapturederror', evt => {
+    assert(false, `WebGPU error: ${evt.error.message}`);
+  });
+
+  ctx.configure({
+    device: t.device,
+    format: navigator.gpu.getPreferredCanvasFormat(),
+    usage: GPUTextureUsage.COPY_DST | GPUTextureUsage.RENDER_ATTACHMENT,
+  });
+
+  const target = document.getElementById('target');
+  t.queue.copyElementImageToTexture(
```

Comment:
Since this is a conformance test, and `copyElementImageToTexture` is not standard yet, the test must pass when `copyElementImageToTexture` doesn't exist. You'll need some JS in both this file AND the ref that does something like show "test skipped" instead of the actual content if `!('copyElementImageToTexture' in GPUQueue)`

Reply:
Added feature detection.